### ISSUE TITLE
Couldn't  able to download the modified dataset

### ISSUE THREAD

**Issue by ParvathyAravindan** [littlebeen/ASDDPM-Adaptive-Semantic-Enhanced-DDPM#17]
Hi, Sorry to bother you again. I am having an issue downloading the modified dataset from Baidu. Can you upload the modified datasets, OLI2MSI and ALSAT, to Huggingface? Many thanks for considering my request.

**Comment by littlebeen** [comment]
OLI2MSI and ALSAT are not proposed by myself. So I couldn't upload those two datasets on my huggingface.

**Comment by ParvathyAravindan** [comment]
When I tested your model with the command,
python test.py --lr_encoder rrdb3 --diffusion_net unetdualfusion --data_train OLI2MSI --data_train_dir OLI2MSI --scale 3 --patch_size 96 
I am getting result images like this.

![Image](https://github.com/user-attachments/assets/30f5432f-3782-4b13-8aed-4bfdab22dfe2)
![Image](https://github.com/user-attachments/assets/8e6fdc37-6b89-4e36-9cf3-2f6013ae199d)
![Image](https://github.com/user-attachments/assets/4c465b9d-7ef9-4a63-b633-ed86d5143ad8)
![Image](https://github.com/user-attachments/assets/cc60e7e3-5b36-4281-be4e-dfbbfca6f13a)

Can you help me with this? Am I loading the pre-trained model incorrectly?

set_hparams(config='./configs/diffsr_alsat4x.yaml',exp_name=args.save,hparams_str="rrdb_ckpt=./pre_train/ASDDPM_OLI2MSI_80001.ckpt") I am using this command for loading the pre-trained model in test.py.

**Comment by littlebeen** [comment]
I have updated the code. Please update by yourself in your own computer. If you load the pretrain model success, it will print 'load success'. If not, it will print 'load fail'. Please check whether it load successfully. By the way, you don't need to change the parameter rrdb_ckpt.

**Comment by littlebeen** [comment]
And it also may because of the dataset. Give me your email, I can send my test set to you by email. Test set is not large.

**Comment by littlebeen** [comment]
I have uploaded some example of the dataset in ./data/OLI2MSI. You can test those image at first and find whether this problem still occurs.

**Comment by ParvathyAravindan** [comment]
Thanks. I will update the code and check with the new dataset.

**Comment by littlebeen** [comment]
Sorry, it is  ./dataset/OLI2MSI not  ./data/OLI2MSI.

**Comment by littlebeen** [comment]
If you find that it is the dataset problem. I can email you all my test set since it is not large.

**Comment by ParvathyAravindan** [comment]
Do I need to update the code in utils/utils.py??

**Comment by littlebeen** [comment]
Yeap, git pull.

**Comment by ParvathyAravindan** [comment]
![Image](https://github.com/user-attachments/assets/a9a02b06-2a91-4bf9-9a73-884d6a12678b) I think the pretrained model is not loaded successfully. Can you tell me in where to check in the code for that?

**Comment by ParvathyAravindan** [comment]
> If you find that it is the dataset problem. I can email you all my test set since it is not large.

I loaded the pretrain model successfully. When I tested the model with the provided OLI2MSI dataset, Its showing a PSNR of 15.67. The output image is attached.  

![Image](https://github.com/user-attachments/assets/56650851-abe5-45a4-9fea-4443a22e1e7d)
![Image](https://github.com/user-attachments/assets/0c684b3d-a266-4091-b18c-03bd12d8def5)

**Comment by littlebeen** [comment]
？Does it work? What's the problem?
